scale/job/serializers.py

```diff
@@ -6,11 +6,11 @@
 import rest_framework.serializers as serializers
 
 from job.models import Job
-from job.job_type_serializers import JobTypeBaseSerializerV5, JobTypeBaseSerializerV6
-from job.job_type_serializers import JobTypeSerializerV5
-from job.job_type_serializers import JobTypeRevisionBaseSerializer
-from job.job_type_serializers import JobTypeRevisionSerializerV5, JobTypeRevisionSerializerV6
-from job.job_type_serializers import JobTypeRevisionDetailsSerializerV6
+# from job.job_type_serializers import JobTypeBaseSerializerV5, JobTypeBaseSerializerV6
```

Comment:
Can these imports be moved back to the top? If not, we need to delete these commented out lines.